internal/test/client/user.go

```diff
@@ -51,3 +52,7 @@ func (c *FakeUserClient) GetClusterCertificate(_ context.Context, cluster, ns st
 func (c *FakeUserClient) GetClusterMetadata(ctx context.Context, ns string) (types.ClusterMetadata, error) {
 	return types.ClusterMetadata{}, nil
 }
+
+func (c *FakeUserClient) GetRegistryCredentials(ctx context.Context, registry string) (dockertypes.AuthConfig, error) {
```

Comment:
```suggestion
func (c *FakeUserClient) GetRegistryCredentials(_ context.Context, _ string) (dockertypes.AuthConfig, error) {
```

shall we make it explicit that those are not used?

I see that the func above is inconsistent too with the functions above.